ColyberCopter/FlightModes/LostRangeFlightMode.h

```diff
@@ -0,0 +1,33 @@
+/**
+ * @file LostRangeMode.h
+ * @author Antoni Wielgus
+ * @brief 
+ * @date 2021-08-28
+ * 
+ */
+
+#ifndef LOSTRANGEMODE_H
+#define LOSTRANGEMODE_H
+
+#include "FlightMode.h"
+
+
+class LostRangeFlightMode : public FlightMode
+{
+    unsigned long startTime;
+    int whenTurnOffMotorsInMiliseconds;
```

Comment:
Always use only: uint8_t, int8_t, uint16_t, int16_t, uint32_t, int32_t, uint64_t, int64_t when you want to be certain about variable size!!!

And change this long name to something much shorter and better expressing what it is for